config/application.yml.default

```diff
@@ -82,6 +82,7 @@ mx_timeout: '3'
 otp_valid_for: '10'
 password_max_attempts: '3'
 personal_key_retired: 'true'
+pii_lock_timeout_in_minutes: '30'
```

Comment:
our entire session only lasts 15 minutes, right? When would this come into play?